src/middle/Middle.ml

```diff
@@ -1,12 +1,37 @@
 include Mir
 open Core_kernel
+module Mir = Mir
```

Comment:
What does this do?

Reply:
Before this change, Mir was being _included_ in Middle.  With this approach, Mir is exposed as a module  which I think is a little nicer since `Middle` contains more than just `Mir`.

As a consequence, you can now open `Middle` and not have all the `Mir` types in scope.

Reply:
Gotcha, thanks!